spec/features/move_classes_spec.rb

```diff
@@ -0,0 +1,92 @@
+require_relative '/Users/emilyfitzsimmons/Documents/projects/servers_and_clients/rps-challenge-apprenticeships/rock.rb'
+require '/Users/emilyfitzsimmons/Documents/projects/servers_and_clients/rps-challenge-apprenticeships/paper.rb'
+
+feature 'Rock is less than' do
+    
+  scenario 'paper' do
+  rock = Rock.new
+  paper = Paper.new
+  expect(rock < paper).to eq true
+  end
+
+  scenario 'scissors' do
+  rock = Rock.new
+  scissors = Scissors.new
+  expect(rock < scissors).to eq false
+  end
+end
+
+feature 'Rock is greater than' do
+    
+  scenario 'paper' do
+  rock = Rock.new
+  paper = Paper.new
+  expect(rock > paper).to eq false
+  end
+
+  scenario 'scissors' do
+  rock = Rock.new
+  scissors = Scissors.new
+  expect(rock > scissors).to eq true
+  end
+
+feature 'Paper is less than' do
+    
+    scenario 'scissors' do
+    paper = Paper.new
+    scissors = Scissors.new
+    expect(paper < scissors).to eq true
+    end
+
+    scenario 'rock' do
+    paper = Paper.new
+    rock = Rock.new
+    expect(paper < rock).to eq false
+    end
+  end
+
+feature 'Paper is greater than' do
+    
+    scenario 'rock' do
+    paper = Paper.new
+    rock = Rock.new
+    expect(paper > rock).to eq true
+    end
+
+    scenario 'scissors' do
+    paper = Paper.new
+    scissors = Scissors.new
+    expect(paper > scissors).to eq false
+    end
+  end
+
+feature 'Scissors is greater than' do
+    
+    scenario 'paper' do
+    scissors = Scissors.new
+    paper = Paper.new
+    expect(scissors > paper).to eq true
+    end
+  
+    scenario 'rock' do
+    scissors = Scissors.new
+    rock = Rock.new
+    expect(scissors > rock).to eq false
+    end
+end
+
+feature 'Scissors is less than' do
+    
+  scenario 'rock' do
+  scissors = Scissors.new
+  rock = Rock.new
+  expect(scissors < rock).to eq true
+  end
+
+  scenario 'paper' do
+  scissors = Scissors.new
+  paper = Paper.new
+  expect(scissors < paper).to eq false
+  end
+end
+end
```

Comment:
I hadn't thought of doing it this way, I like it. I had a long nested if statement that took the two values and returned the outcome. I don't like how mine looks but this is a clever way of doing it with them being there own objects instead of just a variable. Can't think of any tips as I hadn't even considered this method :)

Reply:
I realise this is your tests, 🤦 but my comment stands, I couldn't find a way to test my outcomes because of the random input of ai choice into my method for the game, I couldn't then test the actual mechanic as I couldn't get a test to run not using the random value. nice :)